_data/simple-icons.json

```diff
@@ -954,9 +954,12 @@
 	},
 	{
 		"title": "Android",
-		"hex": "34A853",
+		"hex": "3DDC84",
```

Comment:
> The color value for print is `PMS 2412C` and the online hex color is `#3ddc84`.

`#3DDC84` is the color for print, not for displays.

```suggestion
		"hex": "34A853",
```

![](https://github.com/user-attachments/assets/2771792e-dde9-48b7-bf00-96aacdacf63d)